src/gc/unix/gcenv.unix.cpp

```diff
@@ -318,6 +330,28 @@ bool GCToOSInterface::VirtualRelease(void* address, size_t size)
     return (ret == 0);
 }
 
+// Commit virtual memory range.
+// Parameters:
+//  size      - size of the virtual memory range
+// Return:
+//  Starting virtual address of the committed range
+void* GCToOSInterface::VirtualReserveAndCommitLargePages(size_t size)
+{
+#ifdef HAVE_MAP_HUGETLB
```

Comment:
The reason why this is not working is that it needs to be #if, not #ifdef. The symbol is always defined, only the value is either 0 or 1.

Reply:
Thanks.